spec/fortnox/api/repositories/examples/save_examples.rb

```diff
@@ -0,0 +1,41 @@
+shared_examples_for '.save' do |attribute, fortnox_attribute|
```

Comment:
The `fortnox_attribute` is the same as `described_class.options.json_entity_wrapper`no?